macos/FluentUI/Link/Link.swift

```diff
@@ -122,10 +131,9 @@ open class Link: NSButton {
 }
 
 fileprivate let linkAttributes: [NSAttributedString.Key: Any] = [
-	.foregroundColor: NSColor.linkColor
+	.underlineStyle: NSUnderlineStyle()
```

Comment:
What is this? It just initializes the enum to 0 AKA UnderlineStyleNone? https://developer.apple.com/documentation/uikit/nsunderlinestyle/nsunderlinestylenone?language=objc

Reply:
Right.  I didn't find a corresponding Swift `.none` or anything similar, but this did the trick.